src/bridge/ExposureNotificationAdapter.ios.ts

```diff
@@ -0,0 +1,31 @@
+import {unzip} from 'react-native-zip-archive';
+
+import {ExposureNotification, ExposureConfiguration, ExposureSummary} from './ExposureNotificationAPI';
+
+export default function ExposureNotificationAdapter(
+  exposureNotificationAPI: ExposureNotification,
+): ExposureNotification {
+  return {
+    ...exposureNotificationAPI,
+    detectExposure: async (
+      configuration: ExposureConfiguration,
+      diagnosisKeysURLs: string[],
+    ): Promise<ExposureSummary> => {
+      if (diagnosisKeysURLs.length === 0) {
+        throw new Error('Attempt to call detectExposure with empty list if downloaded files');
+      }
+      const keysZipUrl = diagnosisKeysURLs[0];
```

Comment:
As it is now, the rest will be ignored. 
I guess, the best thing we can do is to change type signature, to take `string` instead of `string[]`.
Native APIs concern me a little though, both iOS and Android leave the room for interpretation. iOS takes array of at least to files (bin and sig) whereas android needs just one file (but is typed such that it can take several files at once). 

I'm not sure whats the best way to handle this

Reply:
Right now I think wee should loop over the array and handle each entry appropriately, but log when we get more than 1, since it would be a surprise at this point.

@henrytao-me what do you think?